src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -97,7 +97,7 @@ export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
               {meta.interval ? (
                 <div className="metaInfoRow">
                   <span className="metaInfoLabel">Interval:</span>
-                  {meta.interval}ms
+                  {formatTimestamp(meta.interval, 4, 1)}
```

Comment:
I think we need 0 instead of 1 if `interval` is an integer. What about:
```suggestion
                  {formatTimestamp(meta.interval, 4, Number.isInteger(meta.interval) ? 0 : 1)}
```

Reply:
I wonder if that should be pulled into `formatTimestamp` itself.